ls1intum/Artemis
# Feature Guided Tour: Modeling exercise tour

### Checklist

### Motivation and Context
This feature should show new Artemis users how to use the modeling editor: https://github.com/ls1intum/Artemis/issues/846

### Steps for Testing
1. Go to the [modeling exercise](https://artemistest.ase.in.tum.de/#/overview/38/exercises/1035) of the `Tutorial Course` on the Test Server
2. The tour should be triggered automatically
3. Navigate through the tour and complete the given tasks
4. Save the changes and start the tutorial again -> the UML model should be deleted

### Screenshots
![m1](https://user-images.githubusercontent.com/33764166/68530888-dbc78b80-030c-11ea-95b5-c07a1c392068.gif)
Reset UML model for tutorial:
![m2](https://user-images.githubusercontent.com/33764166/68530889-e08c3f80-030c-11ea-9383-a135b569ba93.gif)
